lib/Sender.js

```diff
@@ -376,15 +376,10 @@ function getRandomMask () {
  * @private
  */
 function sendFramedData (sender, outputBuffer, data, cb) {
-  try {
```

Comment:
An error is not thrown on the same tick when writing to a closed socket, so I _think_ this only handle the case where `sender._socket` is not a `net.Socket` instance but that never happens as `sender._socket` is never cleared.